src/client.rs

```diff
@@ -144,6 +143,14 @@ impl<T: Transport + 'static> Client<T> {
     }
 }
 
+impl fmt::Debug for ::Client {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "jsonrpc::Client(")?;
+		self.transport.fmt_target(f)?;
+		write!(f, ")")
```

Comment:
nit: indent

Reply:
oh I'll probably have to `:retab` because of borking vim.